hadoop-project/src/site/markdown/index.md.vm

```diff
@@ -15,103 +15,143 @@
+a potential issue of router cache inconsistency due to token loading and renewal. This issue has been addressed by the
+resolution of HDFS-17128.
+
+[HDFS-17148](https://issues.apache.org/jira/browse/HDFS-17148) RBF: SQLDelegationTokenSecretManager must cleanup expired tokens in SQL.
+
+SQLDelegationTokenSecretManager, while fetching and temporarily storing tokens from SQL in a memory cache with a short TTL,
+faces an issue where expired tokens are not efficiently cleaned up, leading to a buildup of expired tokens in the SQL database.
+This issue has been addressed by the resolution of HDFS-17148.
+
+**Others**
+
+Other changes to HDFS RBF include WebUI, command line, and other improvements. Please refer to the release document.
+
+HDFS EC: Code Enhancements and Bug Fixes
+----------------------------------------
 
-```java
-void readVectored(List<? extends FileRange> ranges, IntFunction<ByteBuffer> allocate)
-```
+HDFS EC has made code improvements and fixed some bugs.
 
-All the requested ranges will be retrieved into the supplied byte buffers -possibly asynchronously,
-possibly in parallel, with results potentially coming in out-of-order.
+Important improvements and bugs are as follows:
 
-1. The default implementation uses a series of `readFully()` calls, so delivers
-   equivalent performance.
-2. The local filesystem uses java native IO calls for higher performance reads than `readFully()`.
-3. The S3A filesystem issues parallel HTTP GET requests in different threads.
+**Improvement**
 
-Benchmarking of enhanced Apache ORC and Apache Parquet clients through `file://` and `s3a://`
-show significant improvements in query performance.
+[HDFS-16613](https://issues.apache.org/jira/browse/HDFS-16613) EC: Improve performance of decommissioning dn with many ec blocks.
 
-Further Reading:
-* [FsDataInputStream](./hadoop-project-dist/hadoop-common/filesystem/fsdatainputstream.html).
-* [Hadoop Vectored IO: Your Data Just Got Faster!](https://apachecon.com/acasia2022/sessions/bigdata-1148.html)
-  Apachecon 2022 talk.
+In a hdfs cluster with a lot of EC blocks, decommission a dn is very slow. The reason is unlike replication blocks can be replicated
+from any dn which has the same block replication, the ec block have to be replicated from the decommissioning dn.
+The configurations `dfs.namenode.replication.max-streams` and `dfs.namenode.replication.max-streams-hard-limit` will limit
+the replication speed, but increase these configurations will create risk to the whole cluster's network. So it should add a new
+configuration to limit the decommissioning dn, distinguished from the cluster wide max-streams limit.
 
-Mapreduce: Manifest Committer for Azure ABFS and google GCS
-----------------------------------------------------------
+[HDFS-16663](https://issues.apache.org/jira/browse/HDFS-16663) Allow block reconstruction pending timeout refreshable to increase decommission performance.
```

Comment:
Allow block reconstruction pending timeout refreshable to increase decommission performance.
->
EC: Allow block reconstruction pending timeout refreshable to increase decommission performance.